### PR TITLE
Feat: add CalendarEvent model

### PR DESCRIPTION
**Pull request by AnnaSolovykh**
## Requirements
- Follow the [schema](https://app.diagrams.net/#G14phIB5ljQhlgXhfJa4SJJtoWbJCklcK7#%7B%22pageId%22%3A%22FDPJfdsVZ0Byjb8vGxO2%22%7D) - **2nd version**
- Follow the requirements for validation in [GitBook](https://victorias-organization-2.gitbook.io/kids-first-app) if there are any.

## Test Steps
- Check if the model follows the requirements in schema and gitbook

## Checklist
- [x] Pull Request title includes the issue number and a brief description of the change.
- [x] All changes should be tested and verified to work correctly.
- [x] Code follows backend best practices.
- [x] Branch names follow the conventions in the contributing file.
- [x] Commit messages follow the naming conventions in the contributing file.
- [x] No sensitive data or secrets are included in the codebase.

### CODE REVIEW

**Review by pandjyoti — APPROVED** [review]
Looks perfect

File: models/Calendar.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    type: Schema.Types.ObjectId,
+    ref: 'Kid',
+    required: true,
+  },
+  createdBy: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  sharedWith: {
```
Comment (ElenaCherpakova) [inline models/Calendar.js]: All good to me but I am thinking should it be as an array to allow user sharing event with multiple users?

**Review by ElenaCherpakova — APPROVED** [review]

**Comment by AnnaSolovykh** [comment]
@ElenaCherpakova done! Could you have a look please?

**Comment by ElenaCherpakova** [comment]
Great!

**Pull request merged**